sci-libs/nipype/nipype-9999.ebuild

```diff
@@ -25,6 +25,7 @@ DEPEND="
 	dev-python/numpy[${PYTHON_USEDEP}]
 	sci-libs/nibabel[${PYTHON_USEDEP}]"
 RDEPEND="
+	dev-lang/python[sqlite]
```

Comment:
That's wrong usage of the eclass. Please check the devmanual documentation of the python-r1 eclasses.